BREAKINGCHANGES.md

```diff
@@ -412,9 +436,16 @@ end;
 ---
 
 ## Removed Functionality
-**Error**: _'Codeunit "Type Helper"' does not contain a definition for 'WriteBlob'_\
+**Error**: _'Codeunit "Type Helper"' does not contain a definition for 'GetBlobString'_\
```

Comment:
I don't think using stream functions are the solution to some of these removed functionality.